docs/tutorials/subscription-platform.md

```diff
@@ -485,63 +496,20 @@ Before you begin, make sure you have App Store API credentials set up in the aut
-
-```
-FIRESTORE_EMULATOR_HOST='' pm2 restart auth --update-env
-```
-
-### Put the subscription in Firestore
-
-#### Google IAP
-
-1. In the GCP project, create a new Firestore collection named `${authFirestore.prefix}iap-play-purchases`. You can find the default prefix in `fxa-auth-server/config/index.ts`.
-2. Copy a test Google IAP subscription purchase object from the `fxa-auth-stage-iap-play-purchases` collection in [FxA stage Firestore](https://console.cloud.google.com/firestore/data/fxa-auth-stage-iap-play-purchases?project=moz-fx-fxa-nonprod-375e) as a new document in the new collection.
-3. Manually edit `expiryTimeMillis` to be a date in the distant future.
-4. Create a new FxA user locally and set their user id as the `userId` value.
-
-See [Verify it worked](#verify-it-worked) to confirm everything works correctly.
-
-#### Apple IAP
-
-1. In the GCP project, create a new Firestore collection named `${authFirestore.prefix}iap-app-store-purchases`. You can find the default prefix in `fxa-auth-server/config/index.ts`.
-2. After [QA-1391](https://mozilla-hub.atlassian.net/browse/QA-1391) is resolved, copy a test Apple IAP subscription purchase object from the `fxa-auth-stage-iap-app-store-purchases` collection in [FxA stage Firestore](https://console.cloud.google.com/firestore/data/fxa-auth-stage-iap-play-purchases?project=moz-fx-fxa-nonprod-375e) into the new collection.
-3. Manually edit `expiresDate` to be a date in the distant future and set `status` to `1`.
-4. Create a new FxA user locally and set their user id as the `userId` value.
-
-See [Verify it worked](#verify-it-worked) to confirm everything works correctly.
+    - Enable the IAP integration. See [Google IAP Integration](#google-iap-integration) for Google IAP and [Apple IAP Integration](#apple-iap-integration) for Apple IAP.
+    - Note: For most development, setting `enabled` for both relevant configs is enough without setting up credentials.
+2. Setup a price for IAP
+    - Note: This step is optional, since you can use `examplesku` for a pre-configured Stripe price.
+    - Create a price within Stripe.
+    - Configure the price to have metadata fields `appStoreProductId` and `playSkuId` equal to some unique value (either in Stripe metadata or Firestore, depending). See [Plan Metadata]($stripe-plan-metadata) for these fields.
+3. Create a local fxa user
+    - Grab the id for the user. We'll use this user id for creating mock subscriptions attached to that user.
+4. Run the mock IAP subscription creation script
+    - Within auth-server, run `yarn run create-mock-iap --uid USER_ID --appStoreProductId SKU --playSkuId SKU` replacing USER_ID with the value from step 3, and SKU with the values from step 2.
+    - Omitting the `appStoreProductId` and `playSkuId` flags will use a pre-configured price with the SKU `examplesku` for convenience.
```

Comment:
Most of the time, developers are working on either an Apple IAP ticket or a Google IAP ticket. They don't usually need both fake subscriptions at the same time. Might be worth mentioning that the command can be run with either, both or none at all? As written, this seems to imply you need to concern yourself with both when you may only care about one of them.

Reply:
The script will always create entries for both. This could be changed, but there's no reason not to create both in local imo.